tests/lib/nested.sh

```diff
@@ -1114,6 +1090,14 @@ nested_start_core_vm_unit() {
     fi
     PARAM_IMAGE="$PARAM_IMAGE,${PARAM_PHYS_BLOCK_SIZE},${PARAM_LOGI_BLOCK_SIZE}"
 
+    if nested_is_core_20_system; then
+        # This is to deal with the following qemu error that it just happens using q35 machines in focal
```

Comment:
Maybe reword this a bit, how about
`This is to deal with the following qemu error which occurs using q35 machines in focal`

Reply:
done, thanks for the review